docs/src/prototypes/chatPane/chatPaneContent.tsx

```diff
@@ -24,9 +40,15 @@ class ChatPaneContainer extends React.PureComponent<ChatPaneContainerProps> {
   private generateChatItems(chat: ChatData): JSX.Element[] {
     return generateChatProps(chat).map(({ itemType, ...props }, index) => {
       const ElementType = this.getElementType(itemType)
+      const newProps = { ...props, text: undefined }
       return (
         <Chat.Item key={`chat-item-${index}`}>
-          <ElementType {...props} />
+          {itemType === ChatItemTypes.message && (
+            <div style={ariaLiveStyle} role="heading" aria-level={4}>
```

Comment:
ok, now see where these styles are used - in that case `ariaLiveContainerStyle` or `screenReaderMessageContainer` (even more preferable, to my mind - as it is much more descriptive) would work better